integration_tests/integration/checkAnswers.spec.ts

```diff
@@ -28,13 +29,18 @@ context('Check Answers', () => {
   it('Should contain a full set of correctly formatted move data', () => {
     cy.task('stubExpectedArrival', expectedArrival)
     cy.signIn()
+    cy.setCookie(
```

Comment:
you shouldn't need to set the cookie for these integration tests - just navigating through the application should set and read the application appropriately.

Reply:
Sorted for next commit